hadoop-hdds/docs/content/feature/HA.zh.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "高可用"
+weight: 1
+menu:
+   main:
+      parent: 特性
+summary: Ozone 用于避免单点故障的高可用设置
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone 有两个leader节点（用于键管理的 *Ozone Manager* 和用于块空间管理的 *Storage Container Management* ）以及存储节点（数据节点）。数据是借助 RAFT 共识算法在数据节点之间复制。
+
+为了避免任何单点故障，leader 节点也应该具备高可用的设置。
+
+1. Ozone Manager 的高可用是借助 RAFT 协议实现的；
+2. Storage Container Manager 的高可用则 [还在实现中]({{< ref path="scmha.md" lang="en">}}) 。
+
+## Ozone Manager 的高可用
+
+一个 Ozone Manager 使用 [RocksDB](https://github.com/facebook/rocksdb/) 在本地持久化元数据（卷、桶和键）。 Ozone Manager 的高可用版本在功能上完全一致，只是所有的数据都借助 RAFT 共识算法复制到 Ozone Manager 的 follower 实例上。
+
+![OM HA](HA-OM.png)
+
+客户端连接到 Ozone Manager 上，而 Ozone Manager 负责处理请求并且安排复制。当请求复制到所有的 follower 上后，leader 就可以给客户端回包了。
+
+## 配置
+
+可以在 `ozone-site.xml` 中配置以下设置来启用 Ozone Manager 的高可用模式：
+
+```XML
+<property>
+   <name>ozone.om.ratis.enable</name>
+   <value>true</value>
+</property>
+```
+
+一个 Ozone 的配置（`ozone-site.xml`）支持多个 Ozone 高可用集群。为了支持在多个高可用集群之间进行选择，每个集群都需要一个逻辑名称，该逻辑名称可以解析为 Ozone Manager 的 IP 地址（和域名）。
+
+该逻辑名称叫做 `serviceId`，可以在 `ozone-site.xml` 中进行配置：
+
+ ```
+<property>
+   <name>ozone.om.service.ids</name>
+   <value>cluster1,cluster2</value>
+</property>
+```
+
+对于每个已定义的 `serviceId` ，还应为每个服务器定义一个逻辑配置名：
+
+```XML
+<property>
+   <name>ozone.om.nodes.cluster1</name>
+   <value>om1,om2,om3</value>
+</property>
+```
+
+已定义的前缀可用于定义每个 OM 服务的地址：
+
+```XML
+<property>
+   <name>ozone.om.address.cluster1.om1</name>
+   <value>host1</value>
+</property>
+<property>
+   <name>ozone.om.address.cluster1.om2</name>
+   <value>host2</value>
+</property>
+<property>
+   <name>ozone.om.address.cluster1.om3</name>
+   <value>host3</value>
+</property>
+```
+
+基于 [客户端接口]({{< ref path="interface/_index.md" lang="en">}}) ，定义好的 `serviceId` 就可用于替代单个 OM 主机。
```

Comment:
The Chinese version of the `Interface` document is available, so there is no need to set `lang`.